pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -105,6 +105,25 @@ func (p Provisioner) Provision(options *apibkt.BucketOptions) (*bktv1alpha1.Obje
 	}
 	logger.Infof("set user %q bucket max to %d", p.cephUserName, maxBuckets)
 
+	maxObjects := options.ObjectBucketClaim.Spec.AdditionalConfig["maxObjects"]
+	maxSize := options.ObjectBucketClaim.Spec.AdditionalConfig["maxSize"]
+	if maxObjects != "" || maxSize != "" {
```

Comment:
max objects and size are independent settings, right? Let's check for them individually instead of together. This check for both would only apply to EnableUserQuota(), not all three calls.

Reply:
will address in next PR

Reply:
Done